error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1168,15 +1168,20 @@ Flux<T> after(Flux<T> flux, Predicate<? super T> predicate, Comparator<? super T
   }
 
   /**
-   * Prefer {@link Flux#collect(Collector)} with {@link ImmutableList#toImmutableList} over {@link
-   * Flux#collectList()}.
+   * Prefer {@link Flux#collect(Collector)} with {@link ImmutableList#toImmutableList} over
+   * alternatives.
    */
-  static final class FluxCollectListToCollectImmutableList<T> {
+  static final class FluxCollectToImmutableList<T> {
     @BeforeTemplate
     Mono<List<T>> before(Flux<T> flux) {
       return flux.collectList();
     }
 
+    @BeforeTemplate
+    Mono<List<T>> before2(Flux<T> flux) {
+      return flux.collect(toImmutableList()).map(ImmutableList::copyOf);
```

Comment:
Can we use `Refaster.anyOf` in `before` instead?

Reply:
Ah yeah, same return type so we definitely can 👍🏻.

Reply:
Will push